python/lsst/ap/association/packageAlerts.py

```diff
@@ -77,6 +93,52 @@ def __init__(self, **kwargs):
         self.alertSchema = alertPack.Schema.from_uri(self.config.schemaFile)
         os.makedirs(self.config.alertWriteLocation, exist_ok=True)
 
+        if self.config.doProduceAlerts:
+
+            if confluent_kafka is not None:
+
+                self.password = os.getenv("AP_KAFKA_PRODUCER_PASSWORD")
+                self.username = os.getenv("AP_KAFKA_PRODUCER_USERNAME")
+                self.server = os.getenv("AP_KAFKA_SERVER")
+                self.kafkaTopic = os.getenv("AP_KAFKA_TOPIC")
+
+                if not self.password:
+                    raise ValueError("Kafka password environment variable was not set.")
+                if not self.username:
+                    raise ValueError("Kafka username environment variable was not set.")
+                if not self.server:
+                    raise ValueError("Kafka server environment variable was not set.")
+                if not self.kafkaTopic:
+                    raise ValueError("Kafka topic environment variable was not set.")
```

Comment:
Since you can't do these with a walrus, at least put each `if` directly under the value that is being assigned to.

Reply:
Done